tests/test_default_gaussian.py

```diff
@@ -623,6 +623,8 @@ def test_args(self):
     @pytest.mark.parametrize("g", set(qml.ops.__all_ops__) - set(DefaultGaussian._operation_map.keys()))
     def test_unsupported_gates(self, g, gaussian_device_3_wires):
         """Test error is raised with unsupported gates"""
+        if g == "BasisState":
+            pytest.skip("Test not set up properly for gates with array parameters, needs refactoring.")
```

Comment:
good to leave a TODO here?

Reply:
Will do. This is one of the tests that needs replacing I mentioned above 😆 Too much developer overhead --- adding new independent gates/functions should simply mean new tests, not fixing old tests.

Reply:
done